src/main/java/Greet.java

```diff
@@ -0,0 +1,102 @@
+import java.util.ArrayList;
+
+public class Greet {
+    private static final int indexFix = 1;
```

Comment:
As a constant, perhaps this should be in ["uppercase using underscore to separate words"](https://se-education.org/guides/conventions/java/basic.html#naming)?